### PR TITLE
[es] Views documentation page translated to Spanish

### PR DESCRIPTION
**Pull request by clinchergt**

### CODE REVIEW

File: es/views.rst [hunk 1]
```diff
@@ -1,14 +1,816 @@
...
+diferentes partes. Cada parte tiene distintos usos y será explicada
+en este capítulo:
+
+- **views**: Las vistas son la parte de la página que es única
+  para la acción ejecutándose. Forman la base de la respuesta de la
+  aplicación.
+- **elements**: piezas más pequeñas y reutilizables de código. Los
+  elementos son renderizados típicamente en las vistas.
+- **layouts**: archivos de vistas que contienen code de presentación
+que envuelven muchas interfaces de la aplicación. La mayoría de
```
Comment (antograssiot) [inline es/views.rst]: @clinchergt this line and the next ones needs indentation

File: es/views.rst [hunk 2]
```diff
@@ -1,14 +1,816 @@
...
+simplemente utiliza :term:`plugin syntax`. Si la vista está siendo
+renderizada por un controlador o acción de plugin, el nombre del plugin
+será antepuesto automáticamente a todos los elementos usados, a menos que
+otro nombre de un plugin esté presente. Si el elemento no existe en el
+plugin, buscará en el directorio principal APP::
+
+    echo $this->element('Contacts.helpbox');
+
+Si tu vista es parte de un plugin, puedes omitir el nombre del plugin.
+Por ejemplo, si estás en ``ContactsController` del plugin de Conctacts::
```
Comment (antograssiot) [inline es/views.rst]: missing backtick after ContactsController

File: es/views.rst [hunk 3]
```diff
@@ -1,14 +1,816 @@
...
+-----------------
+
+.. versionadded:: 2.1
+
+Los bloques pueden ser desplegados con el método ``fetch()``. ``fetch()``
+desplegará de forma segura, retornando '' si el bloque no existe::
+
+    echo $this->fetch('sidebar');
+
+``fetch()`` también puede ser usado para desplegar condicionalmente ontenido que debe envolver un bloque si este existe. Esto es útil en layouts o vistas
```
Comment (antograssiot) [inline es/views.rst]: contenido ?

File: es/views.rst [hunk 4]
```diff
@@ -1,14 +1,816 @@
...
+
+El bloque ``content`` contiene los contenidos de la vista renderizada.
+
+``$title_for_layout`` contiene el título de la página. Esta variable es
+generada automaticamente, pero puede ser reasignar como cualquier otra
+variable en tu controlador o vista.
+
+.. note::
+
+    La variable ``$title_for_layout`` estará deperciada a partir de
```
Comment (antograssiot) [inline es/views.rst]: deperciada -> depreciada ?

File: es/views.rst [hunk 5]
```diff
@@ -1,14 +1,816 @@
...
+Los elementos viven en el directorio ``/app/View/Elements``, y tienen
+la extension .ctp. Son desplegados utilizando el método de elementos
+de la vista::
+
+    echo $this->element('helpbox');
+
+Pasar variables a un elemento
+---------------------------------
+
+Puedes pasar datos a un elemento a través del segundo arumento
```
Comment (antograssiot) [inline es/views.rst]: arumento -> argumento

**Comment by clinchergt** [comment]
Hey @antograssiot, thanks for the review.

I've fixed the issues you pointed out.

File: es/views.rst [hunk 6]
```diff
@@ -1,14 +1,817 @@
...
+simplemente utiliza :term:`plugin syntax`. Si la vista está siendo
+renderizada por un controlador o acción de plugin, el nombre del plugin
+será antepuesto automáticamente a todos los elementos usados, a menos que
+otro nombre de un plugin esté presente. Si el elemento no existe en el
+plugin, buscará en el directorio principal APP::
+
+    echo $this->element('Contacts.helpbox');
+
+Si tu vista es parte de un plugin, puedes omitir el nombre del plugin.
+Por ejemplo, si estás en ``ContactsController`` del plugin de Conctacts::
```
Comment (antograssiot) [inline es/views.rst]: @clinchergt you missed one, but was an edit from an existing comment so not obvious... :wink: 
`Contacts` instead of `Conctacts`

**Comment by antograssiot** [comment]
except the last typo, it looks good to me.
Thank you very much for you work @clinchergt :+1:

**Comment by clinchergt** [comment]
@antograssiot: Fixed the last issue you mentioned. I just amended the commit since I didn't want to flood with inane commits, hope that's okay. Thanks again for the review!

**Pull request merged**